frontend/components/ClubTableRow.tsx

```diff
@@ -84,15 +85,18 @@ class ClubTableRow extends React.Component<ClubTableRowProps> {
             <div className="column">
               <div className="columns is-gapless">
                 <div className="column is-4-desktop is-12-mobile">
-                  <Name>{name}</Name>
+                  <Name>
+                    {showElo && club.elo_rank + '. '}
+                    {name}
+                  </Name>
                   <div>
                     <TagGroup tags={tags} />
                   </div>
                 </div>
                 <div className="column is-8-desktop is-12-mobile">
                   <Content>
                     <Subtitle>{this.getSubtitle()}</Subtitle>
-                    <ClubDetails club={club} />
+                    <ClubDetails club={club} showElo={showElo} />
```

Comment:
Is this...
![image](https://github.com/pennlabs/penn-clubs/assets/62971511/73ea1448-0e12-4b7d-ad48-d561a86e0715)

Reply:
![61ZkaCujGrL _AC_UF1000,1000_QL80_](https://github.com/pennlabs/penn-clubs/assets/10285518/b4fa93bc-4592-44d4-a65e-aaa80e411a19)